astronomer/providers/amazon/aws/example_dags/example_redshift_cluster_management.py

```diff
@@ -24,6 +25,18 @@
 ) as dag:
     start = DummyOperator(task_id="start")
 
+    # Execute AWS command then sleep for 5 min so that cluster would be available
+    create_redshift_cluster = BashOperator(
+        task_id="create_redshift_cluster",
+        bash_command="aws redshift create-cluster "
+        "--db-name dev "
```

Comment:
make the db_name astro_dev

Reply:
Changed the db_name from  dev to astro_dev and also parameterize it